_data/tools.yml

```diff
@@ -1272,12 +1272,30 @@
   logo: /assets/images/logos/tableau.png
   logosmall: /assets/images/logos/tableau-small.png
   description: |
-    Tableau is a visual analytics platform transforming the way we use data to solve problems—empowering people and organizations to make the most of their data.
+    Tableau is a visual analytics platform transforming the way we use data to
+    solve problems—empowering people and organizations to make the most of their
+    data.
   links:
     - urltext: Tableau
       url: http://www.tableau.com/
     - urltext: How to connect Tableau to Trino
       url: https://help.tableau.com/current/pro/desktop/en-us/examples_presto.htm
+- name: Testcontainers
```

Comment:
Testcontainers is just one of the libraries Trino uses. Why do we want to document this?